src/constants/commonOptions.js

```diff
@@ -19,6 +19,11 @@ module.exports = {
     placeholder: 'Placeholder',
     tooltip: 'The placeholder text that will appear when this field is empty.'
   },
+  description: {
```

Comment:
Yes, I would like to see this as part of the standard form builder, so if you can provide a setting within the Display tab.